examples/step-60/doc/intro.dox

```diff
@@ -132,8 +132,8 @@ non aligned meshes is complex by itself: to illustrate how is done we study a
 simple problem.
 
 The technique we describe here is presented in the literature using one of many names:
-the **immersed finite element method**, the **fictitious boundary method**, the
-**distributed Lagrange multiplier method**, and others. The main principle is
+the <b>immersed finite element method</b>, the <b>fictitious boundary method</b>, the
+<b>distributed Lagrange multiplier method</b>, and others. The main principle is
```

Comment:
Are these really necessary? Doxygen understands markdown...

Reply:
My copy of doxygen messed up the last one: it just printed `fictitious boundary methods**` for some reason.

Reply:
I see. :)  let's merge then.